.github/ISSUE_TEMPLATE/content_issue.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Doc content issue report
+about: Create a report to help us improve our content.
+title: ''
+
+---
+
+<!--
+**IMPORTANT: this is only for reporting documentation content issues.**
+
+- Report Besu software issus on https://github.com/hyperledger/besu repos.
+- Report doc tool issues using the "Doc tool bug report" template.
+
+**Before filling a bug**, did you try to search in this doc search field? -->
```

Comment:
It's not clear to me what _this_ is referring to here?  If you mean the search field on the doc site, I'd suggest updating this to be more specific. 

Before creating a bug, have you tried using the search field in the documentation to find what you're looking for?